lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -217,6 +220,18 @@ def cmd_reload_lib_tabs(str, words)
     tab_complete_filenames(str, words)
   end
 
+  def cmd_reload_diff_files
+    output = %x(git diff --name-only 2>&1)
+    if output.include?('Not a git repository')
+      print_error 'No git repository found.'
+      return
+    end
+    files = output.split
```

Comment:
What about filenames with whitespace?

Reply:
I hope that's not a thing... I'll split on newline

Reply:
there's some code in metasploit-framework.gemspec that you could check out - it runs everytime metasploit starts that does a similar thing, and handles a few of the edge cases we've discovered over time (and we know it works with everything metasploit actually has)

Reply:
Don't think it's actively a thing, but I'd want to validate my inputs.